app/containers/Settings/Settings.jsx

```diff
@@ -116,17 +132,17 @@ export default class Settings extends Component<Props, State> {
 
   updateExplorerSettings = (e: Object) => {
     const { setBlockExplorer } = this.props
-    setBlockExplorer(e.target.value)
+    setBlockExplorer(e)
   }
 
   updateCurrencySettings = (e: Object) => {
     const { setCurrency } = this.props
-    setCurrency(e.target.value)
+    setCurrency(e)
   }
 
   updateThemeSettings = (e: Object) => {
     const { setTheme } = this.props
-    setTheme(e.target.value)
+    setTheme(e)
   }
 
   deleteWalletAccount = (label: string, key: string) => {
```

Comment:
We can remove this function entirely from Settings as its not going to be used here according to the design spec... Also its implementation exists in wallet manager currently

Reply:
👍

Reply:
Where abouts is this implemented in the wallet manager? Is it not going to be used in settings? This sets the color theme (light/dark) and is currently implemented as a selectInput component (although its designed as a switch in the v2 sketch file)

Reply:
👍